lib/api.ui.js

```diff
@@ -71,6 +71,18 @@ var getSkin = module.exports.getSkin = function(restCtx, callback) {
     RestUtil.RestRequest(restCtx, '/api/ui/skin', 'GET', null, callback);
 };
 
+/**
+ * Retrieve the URL for the logo of a specific tenant.
+ *
+ * @param  {RestContext}        restCtx             Standard REST Context object that contains the current tenant URL and the current user credentials
+ * @param  {Function}           callback            Standard callback method
+ * @param  {Object}             callback.err        Error object containing error code and error message
+ * @param  {String}             callback.css        The URL String for the logo of this tenant.
+ */
+let getLogo = module.exports.getLogo = function(restCtx, callback) {
```

Comment:
Why not `const`? (I think it's better to default to that for things that don't change?)

Reply:
I keep forgetting that `const` can be applied to functions :-/